packages/vela/src/contract-addresses.ts

```diff
@@ -4,11 +4,42 @@ import { Chains } from './utils'
 // Tokens
 const ETHER = zeroAddress
 const ARBITRUM_BTC = '0x2f2a2543b76a4166549f7aab2e75bef0aefc5b0f'
+const ARBITRUM_LTC = '0x48107d739110835dea6f83d8597252764a96b80d'
+const ARBITRUM_ADA = '0xe5cdbed3bc0df7d3637cc78bb6b55ede6d7f3ef5'
+const ARBITRUM_DOGE = '0x5e90df307b07076c42b0516c683c45e10f362c18'
+const ARBITRUM_SHIB = '0x5033833c9fe8b9d3e09eed2f73d2aaf7e3872fd1'
 const ARBITRUM_ARB = '0x912ce59144191c1204e64559fe8253a0e49e6548'
+const ARBITRUM_SOL = '0xa26a142bb3e9d5503b882d17cff43a37b62ee1be'
+const ARBITRUM_MATIC = '0x561877b6b3dd7651313794e5f2894b2f18be0766'
+const ARBITRUM_AVAX = '0x71cb836342a9072d39fa100e2d44a982aff4b54d'
+const ARBITRUM_GBP = '0x1AE27D9068DaDf10f611367332D162d184ed3414'
+const ARBITRUM_EUR = '0x116172B2482c5dC3E6f445C16Ac13367aC3FCd35'
+const ARBITRUM_JPY = '0x95e0e6230e9E965A4f12eDe5BA8238Aa04a85Bc6'
+const ARBITRUM_AUD = '0x7E141940932E3D13bfa54B224cb4a16510519308'
```

Comment:
Most of these will not show up on our frontend, but I dont really see any issues with including all these tokens. 

@Quazia wdyt